newIDE/app/src/ProjectManager/ProjectPropertiesDialog.js

```diff
@@ -39,6 +40,7 @@ type Props = {|
   onApply: (options: { newName?: string }) => Promise<boolean>,
   onPropertiesApplied: (options: { newName?: string }) => void,
   hotReloadPreviewButtonProps?: ?HotReloadPreviewButtonProps,
+  i18n: I18nType,
```

Comment:
If you use i18n as a props, you should remove the i18n provider in your component. 
But it feels a bit hacky to have i18n as a props in a non-UI component.

Reply:
We've actually done that in multiple places, having a mix of i18n passed down as a prop for the functions and used in the render for the jsx